tools/monorepo-utils/src/code-freeze/commands/version-bump/index.ts

```diff
@@ -0,0 +1,118 @@
+/**
+ * External dependencies
+ */
+import { Command } from '@commander-js/extra-typings';
+import simpleGit from 'simple-git';
+
+/**
+ * Internal dependencies
+ */
+import { Logger } from '../../../core/logger';
+import { sparseCheckoutRepoShallow } from '../../../core/git';
+import { octokitWithAuth } from '../../../core/github/api';
+import { getEnvVar } from '../../../core/environment';
+import { getMajorMinor } from '../../../core/version';
+import { bumpFiles } from './bump';
+import { validateArgs } from './lib/validate';
+import { Options } from './types';
+
+const genericErrorFunction = ( err ) => {
+	if ( err.git ) {
+		return err.git;
+	}
+	throw err;
+};
+
+export const versionBumpCommand = new Command( 'version-bump' )
+	.description( 'Bump versions ahead of new development cycle' )
+	.requiredOption( '-v, --version <version>', 'Version to bump to' )
+	.option(
+		'-o --owner <owner>',
+		'Repository owner. Default: woocommerce',
+		'woocommerce'
+	)
+	.option(
+		'-n --name <name>',
+		'Repository name. Default: woocommerce',
+		'woocommerce'
+	)
+	.option(
+		'-b --base <base>',
+		'Base branch to create the PR against. Default: trunk',
+		'trunk'
+	)
+	.action( async ( options: Options ) => {
+		const { owner, name, version, base } = options;
+		Logger.startTask(
+			`Making a temporary clone of '${ owner }/${ name }'`
+		);
+		const source = `github.com/${ owner }/${ name }`;
+		const token = getEnvVar( 'GITHUB_TOKEN' );
+		const remote = `https://${ owner }:${ token }@${ source }`;
+		const tmpRepoPath = await sparseCheckoutRepoShallow(
+			remote,
+			'woocommerce',
+			[
+				'plugins/woocommerce/includes/class-woocommerce.php',
+				// All that's truely needed is the line above, but including these here for completeness.
```

Comment:
Nitpick, you could either fix typo to `truly` or just remove it to improve wording:
```suggestion
				// All that's needed is the line above, but including these here for completeness.
```